static/templates/subscription.hbs

```diff
@@ -2,7 +2,7 @@
 {{#with this}}
 <div class="stream-row" data-stream-id="{{stream_id}}" data-stream-name="{{name}}">
 
-    <div class="check {{#if subscribed }}checked{{/if}} sub_unsub_button {{#unless should_display_subscription_button}}disabled{{/unless}}">
+    <div class="check {{#if subscribed }}checked{{/if}} sub_unsub_button {{#unless should_display_subscription_button}}disabled{{/unless}} tippy-zulip-tooltip" data-tippy-content="{{#if subscribed }}{{t "Click to unsubscribe stream" }}{{else}}{{t "Subscribe to stream" }}{{/if}}">
```

Comment:
Wouldn't the tooltip feel more clean when placed below the `check-icon` ? (Since as of now, it's overlapping with the stream name). 
Here's the gif: 
![tippy1](https://user-images.githubusercontent.com/53977614/115698777-9f61e100-a382-11eb-8043-76c5b65efd35.gif)

Reply:
I guess the top placement might be better. :+1: